src/components/Receive/AddressModal.js

```diff
@@ -36,17 +36,18 @@ class AddressModal extends React.Component<Props, State> {
   /* eslint-disable-next-line react/sort-comp */
   _hideModalTimeoutId = null
 
-  componentWillUnmount() {
-    if (this._hideModalTimeoutId) clearTimeout(this._hideModalTimeoutId)
-  }
-
   async componentDidUpdate() {
     const cbData = await Clipboard.getString()
     if (this.state.isCopied && cbData !== this.props.address) {
+      /* eslint-disable */
```

Comment:
This will disable eslint for the rest of the file. I assume you meant
`// eslint-disable-next-line`